packages/ui/src/components/MessageInput/index.native.tsx

```diff
@@ -179,6 +186,11 @@ export function MessageInput({
                   return;
                 }
 
+                if (e.key === 'Enter' && e.shiftKey) {
+                  window.ReactNativeWebView.postMessage('shift-enter');
```

Comment:
Doesn't hurt, but probably impossible, right? Is shift-enter on mobile a thing?

Reply:
Actually, yeah, we should probably just make enter add the new line rather than have it send, and just do away with shift-enter.